### ISSUE TITLE
Code guidelines

### ISSUE THREAD

**Issue by johndevsvensson** [ezrpg/ezRPG-1.2.x#46]
To keep the project in a maintainable state I suggest we adapt an coding style.

I don't have any particular code style in mind; but one thing I would like to adress is that we omit the closing `?>` from PHP files that only contain PHP. Here's a good reference on why: http://stackoverflow.com/questions/4410704/why-would-one-omit-the-close-tag

**Comment by uaktags** [comment]
I agree on both needing to adopt a coding style and also on the omition of the closing tag. I made mention of this when I started refocusing back on development in the forums:
http://ezrpgproject.net/thread-388.html
